general/g.region/printwindow.c

```diff
@@ -452,15 +537,22 @@ void print_window(struct Cell_head *window, int print_flag, int flat_flag)
                                  PROJECTION_LL);
                 fprintf(stdout, "%-*s %s\n", width, "east-west center:", buf);
             }
+            break;
+        case JSON:
+            json_object_set_number(root_object, "center_easting",
+                                   (window->west + window->east) / 2.);
+            json_object_set_number(root_object, "center_northing",
+                                   (window->north + window->south) / 2.);
+            break;
         }
     }
 
     /* flag.gmt_style */
-    if (print_flag & PRINT_GMT)
+    if ((print_flag & PRINT_GMT) && format != JSON)
```

Comment:
The **json** format should work with GMT and WMS print flags.

Reply:
What should be the output format in case of the GMT and WMS flags used with JSON respectively?

Reply:
The outputs can match what is returned from the plain and shell formats.

```json
{
    "GMT":  "629970/645000/215010/228540",
    "WMS": "bbox=629970,215010,645000,228540"
}
````

Reply:
Done.